### PR TITLE
Now we first check if an element has an uislider already associated. …

### PR DESCRIPTION
**Pull request by showy**
Now we first check if an element has an uislider already associated. If it's the case we destroy that slider first.

Also we moved the assigments in the didInsertElement hook to next sync queue to avoid deprecation warnings.

### CODE REVIEW

**Comment by showy** [comment]
This PR was created because the uislider element wasn't being destroyed after the component was destroyed. So when the component was reinitialized the a new uislider was being created. I'm not sure if this was a special case. Saw another PR related to this problem but it wasn't merged.

Also I was having deprecation warnings using this component, but we can leave out those modifications if necessary.

**Comment by kennethkalmer** [comment]
Thanks so much! Are you refering to #31 as the other PR? The deprecation warnings has also bothered me.

Can I ask that you rebase this as I've merged in #38 which affects the same area of code?

File: addon/components/range-slider.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default Ember.Component.extend({
   behaviour:    'tap',
   tooltips:     false,
 
+
```
Comment (kennethkalmer) [inline addon/components/range-slider.js]: Drop this blank line in the rebase please :)

**Comment by showy** [comment]
Yes I believe it was #31. Sure I'll rebase it.

**Comment by showy** [comment]
Rebased :+1:  Let me know if you see something out of place :)

**Comment by kennethkalmer** [comment]
Thanks @showy! New release going out shortly

**Comment by kennethkalmer** [comment]
v0.12.0 has just been released, thanks for your contributions!

**Pull request merged**